src/App.css

```diff
@@ -36,3 +36,38 @@
     transform: rotate(360deg);
   }
 }
+
+.buttonPanel {
+  width: 720px;
+  background-color: white;
+}
+
+button:hover {
+  border: solid thin #fff;
+}
+
+.group {
+  display: flex;
+}
+
+.group button {
+  width: 100%;
+  height: 100px;
+  font-size: 30px;
+}
+
+.zero button {
+  width: 100%;
+}
+
+.display {
+  width: 700px;
+  height: 100px;
+  background-color: grey;
+  padding: 10px;
+}
+
+.right-side {
+  float: right;
```

Comment:
I think we need to make use of flex all through the styling;
![image](https://user-images.githubusercontent.com/31897434/87582535-6e0d0400-c6d2-11ea-8181-246acdffd126.png)